src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/webdriver/WebElementExt.kt

```diff
@@ -0,0 +1,32 @@
+package com.atlassian.performance.tools.jiraactions.api.webdriver
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import org.openqa.selenium.Keys
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+fun WebElement.sendKeysWhenClickable(driver: WebDriver, vararg keysToSend: CharSequence): WebElement {
+    driver.wait(ExpectedConditions.elementToBeClickable(this))
+    click()
+
+    sendKeys(*keysToSend)
+    return this
+}
+
+fun WebElement.sendKeysAndValidate(driver: WebDriver, text: String): WebElement {
+    sendKeysWhenClickable(driver, text as CharSequence)
+
+    // It's hard to say when the keys can be sent. They seem to randomly get lost.
+    for (i in 0..10) {
+        val actualValue = getHtmlValue() ?: ""
+        if (actualValue == text) {
+            break
+        }
+        Thread.sleep(100)
+        sendKeys(Keys.BACK_SPACE.repeat(actualValue.length), text)
+    }
+    return this
+}
+
+fun WebElement.getHtmlValue(): String? = getAttribute("value")
```

Comment:
In general it's not worth to introduce such shallow wrappers.
Next we'd add `getHref`, `getId`, `getName`, `getType`, `getLabel`, etc.

`getHtmlValue` poses more questions than it answers - what is an HTML value? Is it a `form` value? Is it `innerHTML`?
The answer is: it's an attribute. Using `getAttribute("value")` directly would be clear from the get-go.

- [ ] consider inlining super-thin wrappers